us.zoom.Zoom.json

```diff
@@ -49,6 +47,8 @@
                     "type": "script",
                     "dest-filename": "zoom.sh",
                     "commands": [
+                        "ln -s ~/.var/app/us.zoom.Zoom/config/* ~/.var/app/us.zoom.Zoom/.config",
```

Comment:
This really needs some `ifs` to check whether it already exists or not.

Reply:
I was thinking the very same. But I think it's relying on the directory being empty. In any case, at least a comment would be appropriate.  Because it looks like a gross hack :-/

Reply:
> This really needs some `ifs` to check whether it already exists or not.

Can you submit an MR on the fork? It's my first time hearing about `ifs` and I have no idea what this means :(

Reply:
You are probably looking for https://stackoverflow.com/questions/51539586/how-do-i-use-the-if-then-else-condition-in-json-schema.

Reply:
The problem here is that it makes an optimistic assumption of `~/.var/app/us.zoom.Zoom/config/` being empty and `~/.var/app/us.zoom.Zoom/.config` not existing. It's unlikely to be the case for all users after all changes since Zoom is on Flathub. It should check for edge cases and handle them gracefully instead of creating random symlinks.

Reply:
@barthalion I just wrote a bash script:

```bash
#!/usr/bin/env bash

ZOOMUS=$HOME/.var/app/us.zoom.Zoom/.config/zoomus.conf
if [[ -f "$ZOOMUS" ]]; then
    :
else
    ln -s $HOME/.var/app/us.zoom.Zoom/config/* ~/.var/app/us.zoom.Zoom/.config
fi

ZOOMUS_WAYLAND=$HOME/.var/app/us.zoom.Zoom/.config/zoomus.conf
if [[ -f "$ZOOMUS_WAYLAND" ]]; then
    sed -i 's/enableWaylandShare\=false/enableWaylandShare\=true/' ~/.var/app/us.zoom.Zoom/config/zoomus.conf
else
    echo "File does not exist yet. Re-run Zoom to successfully symlink."
fi
```
but I really didn't understand the link you send, so I can't really do anything here.

Reply:
Should be fixed now https://github.com/flathub/us.zoom.Zoom/pull/217/commits/ad0f11020b4d68048c6c8e74965e30a9f84ce0b8